sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,12 +1,57 @@
 # Release History
 
 ## 4.0.1-beta.4 (Unreleased)
+* Added more samples & enriched docs to `CosmosClientBuilder`. 
+* Updated `CosmosDatabase` & `CosmosContainer` APIs with throughputProperties for autoscale/autopilot support. 
+* Renamed `CosmosClientException` to `CosmosException`. 
+* Replaced `AccessCondition` & `AccessConditionType` by `ifMatchETag()` & `ifNoneMatchETag()` APIs. 
+* Merged all `Cosmos*AsyncResponse` & `CosmosResponse` types to a single `CosmosResponse` type.
+* Renamed `CosmosResponseDiagnostics` to `CosmosDiagnostics`.  
+* Wrapped `FeedResponseDiagnostics` in `CosmosDiagnostics`. 
+* Removed `jackson` dependency from azure-cosmos & relying on azure-core. 
+* Replaced `CosmosKeyCredential` with `AzureKeyCredential` type. 
+* Added `ProxyOptions` APIs to `GatewayConnectionConfig`. 
+* Updated SDK to use `Instant` type instead of `OffsetDateTime`. 
+* Added new enum type `OperationKind`. 
+* Renamed `FeedOptions` to `QueryRequestOptions`. 
+* Added `getETag()` & `getTimestamp()` APIs to `Cosmos*Properties` types. 
+* Added `userAgent` information in `CosmosException` & `CosmosDiagnostics`. 
+* Updated new line character in `Diagnostics` to System new line character. 
 
+## 4.0.1-beta.3 (2020-05-15)
+* Added autoscale/autopilot throughput provisioning support to SDK. 
+* Fixed socket leak issues with Direct TCP client. 
+* Replaced `ConnectionPolicy` with new connection configs. Exposed `DirectConnectionConfig` & `GatewayConnectionConfig` APIs through `CosmosClientBuilder` for Direct & Gateway mode connection configurations.
+* Moved `JsonSerializable` & `Resource` to implementation package. 
+* Fixed `orderByQuery` with continuation token bug.
+* Added `contentResponseOnWriteEnabled` API to CosmosClientBuilder which disables full response content on write operations.
+* Exposed `getETag()` APIs on response types.
+* Moved `CosmosAuthorizationTokenResolver` to implementation. 
```

Comment:
This is  internal change, and we have other similar changes too , do we need to put them all in change log ?

Reply:
Good point @simplynaveen20 . 

Not all of them, just the ones which were being used by customers in previous versions. 
Like `CosmosAuthorizationTokenResolver` was being used by customers. So in case they come to release notes, they should be able to see that we have moved this to implementation.

Reply:
Please suggest more changes like these, so that our customers can check the release notes to see what is supported and what is not in V4.

Reply:
Cool if all these changes related to cx and can be visible to the
them. Just wanted to avoid cx getting overwhelmed with change logs.